### ISSUE TITLE
Red, orange and yellow are very similar

### ISSUE THREAD

**Issue by benlesh** [samccone/moji-brush#29]
Perhaps add the red heart, but leave the others? Or maybe even the heartbreak emoji? Dunno.

Yellow is great... the orange is more of an orange-yellow, which is fine, I think, but it would be nice if there was slightly more range.

Perhaps I'm not using moji-brush as a normal user would though... haha

**Comment by benlesh** [comment]
Let me rephrase this: On Android, the Orange and the Chick are _identical_ in primary color, and there isn't really an orange.

**Comment by samccone** [comment]
we are working on a new palette... coming vry soon

**Comment by joryphillips** [comment]
@samccone @blesh This is the palette direction. I'd like to make it so that: 
1) both sets are available to all platforms; 
2) when tapping a palette color, the user then gets to choose among a selection of these emoji. They can then rotate and resize their selection.

Thoughts? (Those blues are really tough, btw!)

![moij-palette](https://cloud.githubusercontent.com/assets/7508307/15981115/05405a8c-2f26-11e6-98b2-e51af225dc86.png)

**Comment by benlesh** [comment]
Love it!
